external/storm-kafka/src/jvm/storm/kafka/DynamicBrokersReader.java

```diff
@@ -41,6 +41,14 @@
     private String _topic;
 
     public DynamicBrokersReader(Map conf, String zkStr, String zkPath, String topic) {
+        // Check required parameters
+        if(conf == null) {LOG.error("conf cannot be null");}
```

Comment:
Instead putting it in DynamicBrokersReader it will be better if we have this in KafkaConfig and SpoutConfig and have a validate method there.